api/src/main/java/uk/gov/dhsc/htbhf/eligibility/service/EligibilityService.java

```diff
@@ -37,25 +47,38 @@ public EligibilityService(@Value("${dwp.eligibility-check-frequency-in-weeks}")
     public EligibilityResponse checkEligibility(PersonDTO person) {
         LocalDate currentDate = LocalDate.now();
 
-        EligibilityRequest request = EligibilityRequest.builder()
+        DWPEligibilityRequest dwpEligibilityRequest = createDWPRequest(person, currentDate);
+        HMRCEligibilityRequest hmrcEligibilityRequest = createHMRCRequest(person, currentDate);
+
+        DWPEligibilityResponse dwpEligibilityResponse = dwpClient.checkEligibility(dwpEligibilityRequest);
```

Comment:
Mike and I spoke about making these two calls concurrent which should cut the response time to the end user nearly in half. Could be done in a separate commit, though.

We should also bear in mind that when we start making eligibility checks as part of the payment cycle we may want to record which eligibility check was successful, and call that api (DWP/HMRC) first, only calling the other api if we get a response != eligible. Again, not something we have to build now, just bear it in mind.